handler/openid/flow_hybrid.go

```diff
@@ -109,11 +109,14 @@ func (c *OpenIDConnectHybridHandler) HandleAuthorizeEndpointRequest(ctx context.
 		if err := c.AuthorizeExplicitGrantHandler.CoreStorage.CreateAuthorizeCodeSession(ctx, signature, ar.Sanitize(c.AuthorizeExplicitGrantHandler.GetSanitationWhiteList())); err != nil {
 			return errors.WithStack(fosite.ErrServerError.WithCause(err).WithDebug(err.Error()))
 		}
-
-		resp.AddFragment("code", code)
+		if ar.GetRequestForm().Get("response_mode") == "form_post" {
```

Comment:
It looks like the two if statements are copies of one another. Can we avoid duplication here?

Reply:
yes it can be improved